paasta_tools/mesos_maintenance.py

```diff
@@ -200,6 +200,49 @@ def is_host_down(hostname=getfqdn()):
     return hostname in get_down_hosts()
 
 
+def get_hosts_forgotten_draining():
+    """Find hosts that are still marked as draining (rather than down) after the start
+    of their maintenance window. For each of these hosts, either mark them as 'down' for
+    maintenance, adjust the time of their maintenance window, or cancel the maintenance
+    window if it is no longer needed.
+    :returns: a list of hostnames of hosts forgotten draining
+    """
+    draining_hosts = get_draining_hosts()
+    hosts_past_maintenance_start = get_hosts_past_maintenance_start()
+    print("draining_hosts: %s" % draining_hosts)
```

Comment:
Heh, these prints should not have been included, but I can add some `log.debug` calls.